modules/apps/frontend-js/frontend-js-sample-web/src/main/resources/content/Language.properties

```diff
@@ -0,0 +1,2 @@
+javascriptsample.caption=Hello from JavaScript Sample!
```

Comment:
I think this should be javascript-sample-caption. Are you using it?

Reply:
Thanks, and I don't think I'm using the variable...I'll check rq. Will remove if not being used.